lib/libspl/Makefile.am

```diff
@@ -42,6 +42,6 @@ nodist_libspl_la_SOURCES = \
 	$(USER_ASM) \
 	$(KERNEL_C)
 
-libspl_la_LIBADD = -lrt
+libspl_la_LIBADD = -lrt $(TIRPC_LIBS)
```

Comment:
@behlendorf How would you feel about moving xdr.c from libspl to libnvpair? It would avoid forcing libspl consumers from linking to libtirpc. That should mean that we would no longer be doing a strictly faithful reimplementation of OpenSolaris' libc via libspl + glibc, but we would avoid unnecessary dynamic links to libtirpc in things like mount.zfs.